portfolio/src/main/webapp/user-auth/user-auth.template.html

```diff
@@ -0,0 +1,9 @@
+<p ng-show='userAuthCtrl.loading'>Loading...</p> 
+
+<div ng-show='!userAuthCtrl.loading && userAuthCtrl.isLoggedIn()'>
+    <p>Logout <a ng-href= "_ah/logout?continue=%2F"> here </a> </p>
```

Comment:
nit: we only need to use ng-href instead of href when there are data bindings to be evaluated (i.e. stuff inside of curly braces {{userAuthStuff.blah}}). In this scenario we could get away with just href, although ng-href works fine too!